crates/markdown_preview/src/markdown_renderer.rs

```diff
@@ -1,346 +1,319 @@
-use std::{ops::Range, sync::Arc};
-
+use crate::markdown_elements::{
```

Comment:
Most of this file changed so the diff is not helpful. I recommend viewing the file here: https://github.com/kierangilliam/zed/blob/kieran/markdown-parser-and-renderer/crates/markdown_preview/src/markdown_renderer.rs